draft-ietf-quic-http.md

```diff
@@ -1814,6 +1814,15 @@ considerations about exhaustion of stream identifier space apply, though the
 space is significantly larger such that it is likely that other limits in QUIC
 are reached first, such as the limit on the connection flow control window.
 
+QUIC manages stream concurrency on behalf of HTTP/3.  QUIC considers a stream
```

Comment:
I feel this "on behalf of HTTP/3" a bit confusing, because my mental model is that it is generally a responsibility of the transport to manage streams, and that HTTP/2 was the exception.

Maybe something like "In contrast to HTTP/2, stream concurrency in HTTP/3 is managed by QUIC."